admin/src/js/controllers/edit-user.js

```diff
@@ -232,6 +236,27 @@ angular
       return true;
     };
 
+    const validatePlacesPermission = () => {
+      const userRoles = $scope.editUserModel.roles;
+
+      if (!$scope.editUserModel.place || $scope.editUserModel.place.length <= 1) {
```

Comment:
is `$scope.editUserModel.place` expected to always be an array? Are there situations when it can also be a string?

Reply:
No it is always an array. When it was a string, there were situations it was `null` for certain users. Right now I believe it will always be an array even if it is an empty array but for safety I retained `!$scope.editUserModel.place`